projects/plugins/jetpack/modules/related-posts/class.related-posts-customize.php

```diff
@@ -267,15 +275,16 @@ function get_options( $wp_customize ) {
 	 *
 	 * @since 4.4.0
 	 */
-	function customize_controls_enqueue_scripts() {
+	public function customize_controls_enqueue_scripts() {
 		wp_enqueue_script(
 			'jetpack_related-posts-customizer',
 			Assets::get_file_url_for_environment(
 				'_inc/build/related-posts/related-posts-customizer.min.js',
 				'modules/related-posts/related-posts-customizer.js'
 			),
 			array( 'customize-controls' ),
-			JETPACK__VERSION
+			JETPACK__VERSION,
+			$in_footer = false
```

Comment:
I think it can just be `false`:

```suggestion
			false
```